cryostatio/cryostat-web
# fix(labeler): fix labeler config file name

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Fixes: #696 

## Description of the change:
Fix labeler config file to be named correctly.

## Motivation for the change:
This is because it was named like `.labeler.yml` when it should be `labeler.yml`

## How to manually test:
Look at https://github.com/maxcao13/cryostat-web/pull/4 and notice that the action succeeds.
HOWEVER, I'm not sure if it actually works because it doesn't not apply any labels but I'm not sure if that's because it is on a fork. There are some issues within the action labeler repo that talk about how this doesn't work on forks.
